packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -39,57 +46,51 @@ export const getSplitButtonVerticalDividerClassNames = memoizeFunction(
   }
 );
 
-export const getContextualMenuClassNames = memoizeFunction(
```

Comment:
Function is unused and incorrectly marked as `@default` for `IContextualMenuProps#getMenuClassNames`